src/libappimage/desktop_integration/Thumbnailer.cpp

```diff
@@ -78,8 +77,8 @@ namespace appimage {
             } catch (const IconHandleError&) {
                 /* we fail to resize the icon because it's in an unknown format or some other reason
                  * we just have left to write it down as it is and hope for the best. */
-                logger.warning() << "Unable to resize the application icon into a 128x128 image, it will be "
-                                    "written as it's." << std::endl;
+                Logger::warning("Unable to resize the application icon into a 128x128 image, "
```

Comment:
`as it is`, your wording isn't "good style".

Reply:
replaced by "unchanged"

Reply:
`The point was that `it's` was not quite appropriate there (it doesn't "sound good", so to speak), the long form `it is` would've fit. "unchanged" is not quite better, that word doesn't exist AFAICS (maybe "unmodified").